static/css/restyle.less

```diff
@@ -702,6 +704,10 @@ button.good {
     color: #FFF;
 }
 
+.hovercard a h3, .secondary li a, #footer a:link, #footer a:visited {
```

Comment:
Cool, I can do that.

This file is, yeah, a bit of a stylistic mess. I’d like to fix a lot of style nits all in one go in a separate issue. I’ll file an issue for it.